tests/positive/filesystems/skip_readonly.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filesystems
+
+import (
+	"github.com/coreos/ignition/v2/tests/register"
+	"github.com/coreos/ignition/v2/tests/types"
+)
+
+func init() {
+	register.Register(register.PositiveTest, SkipReadOnlyFilesystems())
+}
+
+func SkipReadOnlyFilesystems() types.Test {
+	name := "filesystem.skip.readonly"
+
+	mountPathName := "/usr/share/oem1"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	mntDevices := []types.MntDevice{
+		{
+			Label:        "OEM",
+			Substitution: "$DEVICE",
+		},
+	}
+	config := `{
+		"ignition": {"version": "$version"},
+		"storage": {
+			"filesystems": [{
+				"path": "/dev/sda1",
```

Comment:
This is using `/dev/sda1` as the mountpoint, which could be dangerous if the tests run on a system where `/dev/sda1` is a real disk partition.  It's also not what we need here; the mountpoint should be an existing directory inside a read-only filesystem.

Reply:
I've seen other tests in /positive/filesystems use ```/tmp0``` for the path, will keep this in mind the next time i might have to write a Blackbox test
for now ill drop the negative and positive test